Microsoft.Toolkit.Uwp.SampleApp/SamplePages/RemoteDevicePicker/RemoteDevicePickerCode.bind

```diff
@@ -0,0 +1,8 @@
+﻿RemoteDevicePicker remoteDevicePicker = new RemoteDevicePicker()
+{
+    Title = "Pick Remote Device",
+    SecondaryButtonText = "Select",
+};
+remoteDevicePicker.SelectionMode = bool.Parse(SelectionCheckBox.IsChecked.ToString()) ? ListViewSelectionMode.Extended : ListViewSelectionMode.Single;
```

Comment:
the bool check in the sample code is unnecessary, no need to complicate the example. Simple `remoteDevicePicker.SelectionMode = ListViewSelectionMode.Extended` would do